crates/world/tests/api_test.rs

```diff
@@ -0,0 +1,62 @@
+use bedrockrs_shared::world::dimension::Dimension;
+use bedrockrs_world::level::chunk::{FillFilter, LevelChunkTrait};
+use bedrockrs_world::level::level::default_impl::*;
+use bedrockrs_world::level::level::ChunkSelectionFilter;
+
+#[cfg(feature = "default-impl")]
+#[test]
+fn world_test(
+) -> Result<(), BedrockLevelError<RawInterface, BedrockSubChunkDecoder, BedrockSubChunk>> {
+    let wld_path = std::fs::read_to_string("world_path.txt").ok();
+    if wld_path.is_none() {
+        println!(
+            "Skipping Full Test, Add \"world_path.txt\" next to the Cargo.toml with a world path!"
```

Comment:
Why not just upload an entire world to github for the sake of proper testing?

Reply:
Its commin practice to upload such files if needed for testing, so dw.. example: nbtx